packages/core/src/content/locales.ts

```diff
@@ -1,38 +1,45 @@
 /**
- * Represents the response structure for a query that retrieves a locale.
+ * Represents the "system" field in a locale response.
+ */
+export type SingleLocale = {
```

Comment:
Can it be just `Locale`?
```suggestion
export type Locale = {
```